.github/workflows/node.yml

```diff
@@ -51,6 +51,17 @@ jobs:
         with:
           node-version: ${{ steps.versions.outputs.nodeVersion }}
 
+      - name: Check webpack build changes
+        run: |
+          bash -c "[[ \"`git status --porcelain `\" ]] || (echo 'Do NOT commit changes in js/ as those will be built automatically once merged' && exit 1)"
+
+      - name: Show changes on failure
+        if: failure()
+        run: |
+          git status
+          git --no-pager diff
+          exit 1 # make it red to grab attention
+
```

Comment:
I don't think this does what you think it does.

`git status --porcelain` will not find anything as no compilation was run yet.

I see two alternatives:
* `git diff {name of base branch} js` and then fail if it has some output
* use the github mechanism to run tasks on changed files in js. (Maybe even in a separate task)

Reply:
Yes indeed. I moved to a separate workflow and used https://github.com/tj-actions/changed-files which uses the API instead of a local git clone to keep the action simple.